insights.md

```diff
@@ -10,10 +11,9 @@
 * Optimal time for maintenance/upgrades
 
 ## Take a Tour of DevOptics Run Insights
-1. Go to https://devoptics.cloudbees.com in your browser and login with the email **beedemo.guest@gmail.com** and the password provided by your instructor <p><img src="img/insights/devoptics_login.png" width=800/>
-2. Click on the graph icon in the left menu to switch to the Run Insights view <p><img src="img/insights/insights_view.png" width=800/>
-3. Next click on the **?** left menu item and then select **Tour Run Insights** from the pop-up menu <p><img src="img/insights/tour_run_insights.png" width=800/>
-4. Complete the tour and then continue to the next exercise <p><img src="img/insights/start_tour.png" width=800/>
+Run insights provide you with a real-time and historical view of job and pipeline activity. You can connect any number of masters for an aggregate view across the organization. Masters can be a heterogeneous mix of CloudBees Core and Jenkins. Metrics are viewable over an adjustable time period.
```

Comment:
CloudBees Core Masters and OSS Jenkins masters

Reply:
Updated - CloudBees Core Masters and Open Source Jenkins masters